src/Symfony/Component/Mime/Resources/bin/update_mime_types.php

```diff
@@ -14,7 +14,7 @@
 }
 
 // load new map
-$data = json_decode(file_get_contents('https://cdn.jsdelivr.net/gh/jshttp/mime-db@v1.49.0/db.json'), true);
+$data = json_decode(file_get_contents('https://cdn.jsdelivr.net/gh/jshttp/mime-db@v1.52.0/db.json'), true);
```

Comment:
is there a way to make it use the latest version each time the script is run (as done for the freedesktop data below where we use the master branch) ?

Reply:
`https://cdn.jsdelivr.net/gh/jshttp/mime-db/db.json`?